eclair-core/src/main/scala/fr/acinq/eclair/Eclair.scala

```diff
@@ -18,7 +18,7 @@ package fr.acinq.eclair
 
 import akka.actor.ActorRef
 import akka.actor.typed.scaladsl.AskPattern.Askable
```

Comment:
This should remove the need for the boilerplate.

```suggestion
import akka.actor.typed.scaladsl.AskPattern.{Askable, schedulerFromActorSystem}
```